src/Shuffle/lib.mo

```diff
@@ -15,6 +15,7 @@ module {
     *********/
 
     var _isShuffled = false;
+    let minIndex = if (config.legacyPlaceholder == ?true) 1 else 0;
```

Comment:
in theory we can ignore the legacy placeholder stuff here, right? because the collections that have already been deployed with this old code will not be shuffled anymore